plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -155,6 +155,55 @@ public void testReadWholePartition()
         assertUpdate("DROP TABLE test_read_part_key");
     }
 
+    @Test
+    public void testHistorySystemTable()
```

Comment:
Please add to test `io.trino.plugin.deltalake.TestDeltaLakeSystemTables` also a test verifying the accuracy of the filters on `$history` table.